cloudify_agent/shell/commands/daemons.py

```diff
@@ -21,6 +21,8 @@
 from cloudify_agent.api.factory import DaemonFactory
 from cloudify_agent.shell import env
 from cloudify_agent.shell.decorators import handle_failures
+from cloudify.context import BootstrapContext
```

Comment:
these two new imports are cloudify related 3rd party so they should some between to currently two existing sections